scripts/game_structure/load_cat.py

```diff
@@ -121,6 +121,7 @@ def json_load():
             new_cat.adoptive_parents = cat["adoptive_parents"] if "adoptive_parents" in cat else []
             
             new_cat.genderalign = cat["gender_align"]
+            new_cat.pronouns = cat["pronouns"]
```

Comment:
This causes a load-error when for older saves.  To fix this issue, use `.get` , with an appropriate default.

Reply:
got this done! One of the easier things.